packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -102,6 +104,7 @@ const ContextMenuTrigger = React.forwardRef<ContextMenuTriggerElement, ContextMe
       []
     );
     const handleOpen = (event: React.MouseEvent | React.PointerEvent) => {
+      if (disabled) return;
```

Comment:
I think that this should be moved outside of this handler to event handlers. The problem with the current implementation is that it still prevents the default in the `onContextMenu` handler and thus it prevents the native context menu to open.

I've prepared a PR with this suggestion implemented: https://github.com/paperstick/primitives/pull/1

Reply:
Thanks, @Andarist, a good suggestion! I have just cleaned up the code a bit and merged the PR, thanks for the contibution!